wikipron/extract/default.py

```diff
@@ -0,0 +1,31 @@
+"""Default word and pron extraction."""
+
+import itertools
+import typing
+
+import requests
+
+from wikipron.extract.core import yield_pron
+
+
+if typing.TYPE_CHECKING:
+    from wikipron.config import Config
+    from wikipron.typing import Iterator, Pron, Word, WordPronPair
+
+
+IPA_XPATH = '//span[@class = "IPA"]'
+
+
+def _yield_phn(
+    request: requests.Response, config: "Config"
+) -> "Iterator[Pron]":
+    for pron_xpath in request.html.xpath(config.pron_xpath_selector):
+        yield from yield_pron(pron_xpath, IPA_XPATH, config)
```

Comment:
Comparing the default treatment and Khmer, one common denominator is what I've abstracted out as the `yield_pron` function (defined in `wikipron/extract/core.py`. The default case needs an extra "layer" of looping over what we previously called the "li_selector".

`yield_pron` should hopefully be generic enough and usable by other exceptional languages.

Why isn't there a `yield_word`? Both default and Khmer simply use the head word from a Wiktionary entry page, and therefore no special treatment is needed. Latin would likely need its own `yield_word`, for grabbing the desired graphemes with macrons.